main/src/main/java/cgeo/geocaching/maps/MapUtils.java

```diff
@@ -424,8 +426,6 @@ public void onSlide(@NonNull final View sheet, final float slideOffset) {
             };
 
             b.addCallback(callback);
-            swipeToOpenFragment.setOnStopCallback(() -> b.removeCallback(callback));
```

Comment:
It's important to remove the callback as soon as the popup closes. See #14868. If it cannot be done via the swipeToOpenFragment, we need some other callback.

Reply:
I've updated the PR to remove the callback on closing the popup